src/js/interfaces/coordinateForm.ts

```diff
@@ -0,0 +1,36 @@
+export interface SpecificDMSSection {
+  rowNum: number;
+  latitude: {
+    degree: string;
+    minutes: string;
+    seconds: string;
+    cardinalPoint: string;
```

Comment:
Can we do something like this? 
![image](https://user-images.githubusercontent.com/12076555/74879543-599d2900-5337-11ea-92d3-332bdd33ed47.png)
![image](https://user-images.githubusercontent.com/12076555/74879552-5efa7380-5337-11ea-86a2-ab6c949d5a5c.png)